hack/consecutive-upgrades-test.sh

```diff
@@ -0,0 +1,267 @@
+#!/bin/bash -e
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2023 Red Hat, Inc.
+#
+# Usage:
+# make upgrade-test-operator-sdk
+#
+# Use Openshift-CI "optional-operators-ci-*" workflow to:
+# - Use the operator-sdk to upgrade a pre deployed bundle; use a new bundle,
+#   named 100.0.0 with the contents of the open PR (this can include new
+#   dependent images, new CRDs...).
+# - the script then performs various validations against the upgraded version.
+
+MAX_STEPS=$(( $(grep -c "Msg " "$0") - 2)) # subtract self line and the function name
+CUR_STEP=1
+HCO_DEPLOYMENT_NAME=hco-operator
+HCO_WH_DEPLOYMENT_NAME=hco-webhook
+HCO_NAMESPACE="kubevirt-hyperconverged"
+HCO_KIND="hyperconvergeds"
+HCO_RESOURCE_NAME="kubevirt-hyperconverged"
+MID_VERSION=1.11.0
+TARGET_VERSION=100.0.0
+VMS_NAMESPACE=vmsns
+
+OUTPUT_DIR=${OUTPUT_DIR:-_out}
+OO_MID_BUNDLE=${OO_MID_BUNDLE}
+OO_LAST_BUNDLE=${OO_LAST_BUNDLE}
+
+echo "INITIAL_VERSION: $INITIAL_VERSION"
+
+function Msg {
+    { set +x; } 2>/dev/null
+    echo "--"
+    for a in "$@"; do
+        echo "Upgrade Step ${CUR_STEP}/${MAX_STEPS}: $a"
+    done
+    echo "--"
+    ((CUR_STEP += 1))
+    set -x
+}
+
+export CMD="oc"
+
+echo "oc version"
+${CMD} version || true
+
+function cleanup() {
+    rv=$?
+    if [ "x$rv" != "x0" ]; then
+        echo "Error during upgrade: exit status: $rv"
+        make dump-state
+        echo "*** Upgrade test failed ***"
+    fi
+    exit $rv
+}
+
+function upgrade() {
+  I_VERSION=$1
+  T_VERSION=$2
+  BUNDLE=$3
+
+  source ./hack/check-uptime.sh
+  sleep 5
+  INITIAL_BOOTTIME=$(check_uptime 10 60)
+
+  Msg "HCO deployOVS annotation and OVS state in CNAO CR before the upgrade"
```

Comment:
This is common to all the `Msg` calls in this function:

Let's add some text to distinguish between the two calls to this function. So we could find from the message what upgrade this log is related to (e.g. `[va.b.c => vx.y.z]` or any other text that will mark where we are in the process.